Vagrantfile

```diff
@@ -0,0 +1,67 @@
+# shared token for joining kubernetes cluster
+$cluster_token = "cxie6r.0gxsw80zf2xcti1d"
+
+# global settings for all boxes
+$box = "ubuntu/xenial64"
+
+# master settings
+$master_hostname = "kube-master"
+$master_ip = "192.168.64.2"
+$master_cpus = 2 # will run with less, this just makes provisioning quicker
+$master_memory = 2048 # will run with less, this just makes provisioning quicker
+
+# worker settings
+$node_count = 3
+$node_base_hostname = "kube-node"
+$node_base_ip = "192.168.64."
+$node_base_ip_cni = "10.128."
+$node_cpus = 2 # will run with less, this just makes provisioning quicker
+$node_memory = 2048 # will run with less, this just makes provisioning quicker
+
+# configs
+Vagrant.configure("2") do |config|
+  config.vm.define "master" do |master|
+    master.vm.box = $box
+    master.vm.hostname = $master_hostname
+    master.vm.network "private_network", ip: $master_ip
+    master.vm.synced_folder "config", "/config"
+    master.vm.provider "virtualbox" do |v|
```

Comment:
@webvictim would you be able to also add `libvirt` as a provider, please?
Many of us tend to stick to that as opposed to use `virtualbox`, nowadays.
Thanks

Reply:
@eldios Sure - I'm just testing out the changes as I've had to use a different base image for libvirt. I should be able to commit the updated code shortly.

Reply:
thanks.